packages/l1-l3-teleport/.env-sample

```diff
@@ -0,0 +1,14 @@
+# L3 Node
+
+L3RPC=""
+
+# Hosted Aggregator Node (JSON-RPC Endpoint). This is Arbitrum Sepolia Testnet, can use any Arbitrum chain
+
+L2RPC="https://sepolia-rollup.arbitrum.io/rpc"
+
+# Ethereum RPC; i.e., for Sepolia https://sepolia.infura.io/v3/<your infura key>
+
+L1RPC=""
+
+# Private Key
+L1PRIVKEY=""
```

Comment:
Can we use `DEVNET_PRIVKEY` here? I know it's not very accurate, but it's the same that's used accross all packages, so it would prevent using 2 private keys in the root-level .env file.

We can refactor some variable names in a future PR.

Reply:
727db9055f2cbb8e89d76367d53501070d62dcde